src/main/java/xyz/nucleoid/plasmid/game/GameWorld.java

```diff
@@ -144,13 +207,26 @@ public boolean containsEntity(LivingEntity entity) {
 
     @Nonnull
     public <T> T invoker(EventType<T> event) {
-        return this.listeners.invoker(event);
+        return this.game.getListeners().invoker(event);
     }
 
+    /**
+     * Tests whether the given {@link GameRule} passes in this {@link GameWorld}.
+     *
+     * <p>As an example, calling this method with {@link GameRule#BLOCK_DROPS} will return a {@link RuleResult} that describes whether blocks can drop.
+     *
+     * @param rule the {@link GameRule} to test in this {@link GameWorld}
+     * @return a {@link RuleResult} that describes whether the {@link GameRule} passes
+     */
     public RuleResult testRule(GameRule rule) {
-        return this.rules.test(rule);
+        return this.game.getRules().test(rule);
     }
 
+    /**
+     * Attempts to tick this {@link GameWorld} and any {@link GameTickListener#EVENT} listener attached to it.
+     *
+     * <p>If this {@link GameWorld} is closed ({@link GameWorld#closed}), the tick will not execute.
```

Comment:
I think it's not important to mention the internal field :)
```suggestion
     * <p>If this {@link GameWorld} has been closed, the tick will not execute.
```

Reply:
Thanks. My initial intention was to show how to check whether the world is closed already, but the field is private anyway, so it doesn't matter.